lib/internal/modules/esm/load.js

```diff
@@ -1,8 +1,67 @@
 'use strict';
 
 const { defaultGetFormat } = require('internal/modules/esm/get_format');
-const { defaultGetSource } = require('internal/modules/esm/get_source');
 const { validateAssertions } = require('internal/modules/esm/assert');
+const {
+  ArrayPrototypeConcat,
+  RegExpPrototypeExec,
+  decodeURIComponent,
+} = primordials;
```

Comment:
Can we put this at the top of the files, for consistency with other internal modules?